pyproject.toml

```diff
@@ -1,44 +1,63 @@
-[tool.poetry]
+[build-system]
+requires = ["setuptools >= 42", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
 name = "turbomoleio"
-version = "1.4.0"
 description = """\
 Turbomoleio is a python package containing a set of tools \
 for the generation of inputs and parsing of outputs for \
 TURBOMOLE."""
 readme = "README.rst"
-authors = ["davidwaroquiers <david.waroquiers@matgenix.com>"]
-license = "GPL V3"
-#packages = [
-#    { include = "turbomoleio", from = "src" }
-#]
-
-[tool.poetry.dependencies]
-python = "^3.8.12"
-numpy = "^1.16.5"
-matplotlib = "^3.5.1"
-pandas = "^1.4.1"
-pymatgen = ">=2022.2.10"
-monty = ">2022.1.19"
-pexpect = "^4.8.0"
-cerberus = "^1.3.4"
+keywords = []
+license = { text = "GPL V3" }
+authors = [
+    { name = "Guido Petretto", email = "guido.petretto@matgenix.com" },
+    { name = "David Waroquiers", email = "david.waroquiers@matgenix.com" }
+]
+version = "1.5.0"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "Intended Audience :: System Administrators",
+    "Intended Audience :: Information Technology",
+    "Operating System :: OS Independent",
+    "Topic :: Other/Nonlisted Topic",
+    "Topic :: Scientific/Engineering",
+]
+requires-python = ">=3.9"
+dependencies =[
+    "numpy >= 1.16.5",
+    "matplotlib >= 3.5.1",
+    "pandas >= 1.4.1",
+    "pymatgen >= 2022.2.10",
```

Comment:
Does everything work with the old 2022.2.10 version? The tests here run only with the latest. I suppose we should keep this for compatibility with old workflows

Reply:
no you're right and it probably does not work with that older version... I'll check and update to a more recent that works.

Reply:
Well it seems it's working. So I'll keep it with 2022.